azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/__init__.py

```diff
@@ -23,31 +32,51 @@ def configure_azure_monitor(**kwargs):
     """
 
     configurations = get_configurations(**kwargs)
-    disable_tracing = configurations.get("disable_tracing", False)
+    connection_string = configurations.get("connection_string", False)
     service_name = configurations.get("service_name", "")
     service_namespace = configurations.get("service_namespace", "")
     service_instance_id = configurations.get("service_instance_id", "")
+    disable_logging = configurations.get("disable_logging", False)
+    logging_level = configurations.get("logging_level", NOTSET)
+    disable_tracing = configurations.get("disable_tracing", False)
     sampling_ratio = configurations.get("sampling_ratio", 1.0)
     tracing_export_interval_millis = configurations.get(
         "tracing_export_interval_millis", 30000
     )
 
-    if not disable_tracing:
+    resource = None
+    if not disable_logging or not disable_tracing:
         resource = Resource.create(
             {
                 ResourceAttributes.SERVICE_NAME: service_name,
                 ResourceAttributes.SERVICE_NAMESPACE: service_namespace,
                 ResourceAttributes.SERVICE_INSTANCE_ID: service_instance_id,
             }
         )
+    if not disable_logging:
+        logging_provider = LoggerProvider(resource=resource)
+        log_exporter = AzureMonitorLogExporter(
+            connection_string=connection_string
+        )
+        logging_provider.add_log_record_processor(
+            BatchLogRecordProcessor(log_exporter)
+        )
+        handler = LoggingHandler(
+            level=logging_level, logger_provider=logging_provider
+        )
+        getLogger().addHandler(handler)
+        set_logger_provider(logging_provider)
```

Comment:
Nit: Just to be consistent, call `set_logger_provider` immediately after provider instantiation.

Reply:
Makes more sense to me this way since there remains more configuration to be done with the provider

Reply:
Configuration can still be done to the provider after setting it globally. We've had interesting bugs before in which user's were not setting the providers (or could not) so then they were just doing no-ops. Setting provider should be the first thing that user does.

Reply:
Alright. Will change

Reply:
Done